src/Spd.Manager.Licence/BizPortalUserManager.cs

```diff
@@ -54,6 +55,38 @@ public async Task<BizPortalUserResponse> Handle(BizPortalUserCreateCommand reque
         return _mapper.Map<BizPortalUserResponse>(response);
     }
 
+    public async Task<BizPortalUserResponse> Handle(BizPortalUserUpdateCommand request, CancellationToken ct)
+    {
+        PortalUserListResp existingUsersResult = await _portalUserRepository.QueryAsync(
+            new PortalUserQry()
+            {
+                OrgId = request.BizPortalUserUpdateRequest.BizId,
+                ContactRoleCode = new List<ContactRoleCode> { ContactRoleCode.PrimaryBusinessManager, ContactRoleCode.BusinessManager },
+                PortalUserServiceCategory = PortalUserServiceCategoryEnum.Licensing
+            },
+            ct);
+
+        //check if email already exists for the user
+        if (existingUsersResult.Items.Any(u => u.UserEmail != null && u.UserEmail.Equals(request.BizPortalUserUpdateRequest.Email, StringComparison.InvariantCultureIgnoreCase)))
```

Comment:
I got confused, if user does not update his email, for example, he just want to update role, then there is exception throws?

Reply:
Good stuff you asked that, I didn't consider this use case properly. I added a condition so the logic checks the duplicated email in "other users" instead of the current one. Ty